samza-core/src/main/java/org/apache/samza/task/AsyncRunLoop.java

```diff
@@ -611,10 +611,12 @@ public void run() {
             List<TaskCallbackImpl> callbacksToUpdate = callbackManager.updateCallback(callbackImpl);
             for (TaskCallbackImpl callbackToUpdate : callbacksToUpdate) {
               IncomingMessageEnvelope envelope = callbackToUpdate.envelope;
-              log.trace("Update offset for ssp {}, offset {}", envelope.getSystemStreamPartition(), envelope.getOffset());
+              log.trace("Update offset for ssp {}, checkpoint offset {}", envelope.getSystemStreamPartition(),
```

Comment:
log.trace("Update offset for ssp {}, offset {}, checkpoint offset {}", envelope.getSystemStreamPartition(), envelope.getOffset(), envelope.getCheckpointOffset());